recipes-kernel/arp-driver/arp-driver_git.bb

```diff
@@ -0,0 +1,14 @@
+SUMMARY = "ARP driver kernel module"
+DESCRIPTION = "Automotive Reference Platform Kernel module"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=b234ee4d69f5fce4486a80fdaf4a4263"
+PV = "1.0+git${SRCREV}"
+PR = "r0"
+
+inherit module
+
+SRCREV = "96d8815cd1dc6a47e091d2865939e62c132caf1d"
+
+SRC_URI = "git://github.com/TSAnsari/arp-driver.git;protocol=https;branch=master"
```

Comment:
Why would the code be hosted on your private fork? Also the SRCREV doesn't seems to exist.